SETUP.md

```diff
@@ -98,6 +98,7 @@ External contributors can supply alternate placeholder values for secrets normal
 ```
 slack_bot_token: localoverride
 pardot_private_key: localoverride
+openai_student_learning_api_key: localoverride
```

Comment:
Remove this (accidentally committed?)

Reply:
Removed in [b2320b2](https://github.com/code-dot-org/code-dot-org/pull/64770/commits/b2320b29c783ff28f4d5a53dfcfabd6c8c699a32).